awsauth.go

```diff
@@ -267,6 +268,22 @@ func GetCredentials(c *Config) (*awsCredentials.Credentials, error) {
 		log.Print("[INFO] ECS container credentials detected, RemoteCredProvider added to auth chain")
 	}
 
+	if filePath := os.Getenv("AWS_WEB_IDENTITY_TOKEN_FILE"); len(filePath) > 0 {
```

Comment:
`AWS_WEB_IDENTITY_TOKEN_FILE` is come here.

https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/
> Because the service account has an eks.amazonaws.com/role-arn annotation, the webhook injects the necessary environment variables (AWS_ROLE_ARN and AWS_WEB_IDENTITY_TOKEN_FILE) and sets up the aws-iam-token projected volume in the pod that the job supervises.